config.json

```diff
@@ -14,6 +14,18 @@
         "initiateLifetime": {
             "hours": 1
         },
+        "loginDir": "/login/",
+        "loginHash": "sha512",
+        "loginIdLength": 24,
+        "loginLifetime": {
+            "minutes": 15
+        },
+        "passwordHash": {
+            "algo": "sha512",
+            "hashLength": 48,
+            "iterations": 100000,
+            "salt": ""
```

Comment:
This should be the length of the salt, not the salt itself.  We should generate a random salt for each account.